10-Summation_of_primes.js

```diff
@@ -0,0 +1,32 @@
+// The sum of the primes below 10 is 2 + 3 + 5 + 7 = 17.
+//
+// Find the sum of all the primes below two million.
+
+(function primesSum() {
+  var maxPrimeValue = 2e+6;
+  var i = 2;
+  var sum = 0;
+
+  while(i <= maxPrimeValue) {
+    if(isPrime(i) === true) {
+      sum += i;
+    }
+    i++;
+  }
+
+  function isPrime(n) {
+    //x is the largest value could be a prime number
+    var x = Math.floor(Math.sqrt(n));
+
+    while (x >= 2) {
+      if (n % x === 0) {
+        return false;
+      }
+      x--;
+    }
+    return true;
+  }
+
+  console.log(sum);
+  return sum;
+}());
```

Comment:
Next time please declare the function first, which returns the value, then call it in the main flow, catch the result and log it out.